mycore-base/src/main/java/org/mycore/common/config/MCRConfigurableInstanceHelper.java

```diff
@@ -114,33 +111,78 @@ public static <T> Optional<T> getInstance(String name) throws MCRConfigurationEx
      * @throws MCRConfigurationException if the property is not right configured.
      */
     public static <S> Optional<S> getInstance(Class<S> superClass, String name) throws MCRConfigurationException {
+        return getInstance(superClass, name, false);
+    }
+
+    /**
+     * Creates a configured instance of a class.
+     *
+     * @param superClass the intended super class of the instantiated class
+     * @param name       the property which contains the class name
+     * @return the configured instance of T
```

Comment:
Please describe third param `allowMissingClassNameForFinalClasses`
The paramName is rather long, but I don't have a better idea